docs/openfaas-pro/iam/function-authentication.md

```diff
@@ -0,0 +1,272 @@
+# Function Authentication
```

Comment:
Link to this page from the older authentication page

https://docs.openfaas.com/reference/authentication/

Above webhooks:

```
## IAM Based Authentication

...
```